apps/dotcom/client/src/tla/components/TlaFileMenu/TlaFileMenu.tsx

```diff
@@ -146,12 +146,14 @@ export function FileItems({
 	}, [app, fileId, groupId])
 
 	const handleDuplicateClick = useCallback(async () => {
+		if (!groupId) return
 		const newFileId = uniqueId()
 		const file = app.getFile(fileId)
 		if (!file) return
 		trackEvent('duplicate-file', { source })
 		const res = await app.createFile({
 			fileId: newFileId,
+			groupId,
```

Comment:
Without this the duplicated files were always getting created it the home group.